plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -172,10 +176,14 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             ((==) @Integer)
             (runCostingFunTwoArguments . paramEqualsInteger)
-    toBuiltinMeaning Concatenate =
+    toBuiltinMeaning AppendByteString =
         makeBuiltinMeaning
             BS.append
-            (runCostingFunTwoArguments . paramConcatenate)
+            (runCostingFunTwoArguments . paramAppendByteString)
+    toBuiltinMeaning ConsByteString =
+        makeBuiltinMeaning
+            (\n xs -> BSC.cons (chr $ fromIntegral @Integer n) xs)
```

Comment:
I don't think we need to go via the `Char8` interface, do we? Can't we just use https://hackage.haskell.org/package/bytestring-0.10.8.2/docs/Data-ByteString.html#v:cons ?

Reply:
Yep, missed this one. Fixed. Thanks!